src/workers/notifications/userGiftedPlusNotification.ts

```diff
@@ -0,0 +1,70 @@
+import { NotificationType } from '../../notifications/common';
+import { generateTypedNotificationWorker } from './worker';
+import { NotificationGiftPlusContext } from '../../notifications';
+import { SquadSource, User, UserSubscriptionFlags } from '../../entity';
+import { isGiftedPlus, isPlusMember } from '../../paddle';
+import { queryReadReplica } from '../../common/queryReadReplica';
+import { PLUS_MEMBER_SQUAD_ID } from '../userUpdatedPlusSubscriptionSquad';
+import { CioTransactionalMessageTemplateId, sendEmail } from '../../common';
+
+async function sendNotificationEmailToGifter(ctx: NotificationGiftPlusContext) {
+  const message_data = {
+    recipient_name: ctx.recipient.name,
+    recipient_image: ctx.recipient.image,
+  };
+
+  await sendEmail({
+    send_to_unsubscribed: true,
+    identifiers: { id: ctx.gifter.id },
+    transactional_message_id:
+      CioTransactionalMessageTemplateId.UserSentPlusGift,
+    to: ctx.gifter.email,
+    message_data,
+  });
+}
+
+const worker = generateTypedNotificationWorker<'user-updated'>({
```

Comment:
Probably weird one, but this should also receive when you blocked this user? 😅

Reply:
Yes, I think that this case is not covered here. Should I mute notification for blocked users? This will result in a very odd case 🤔

Reply:
Technically I think blocked users should not show in the gift modal right?

Reply:
> Maybe is better to disable users that blocked you, from the select dropdown on frontend, if we want to handle this case. Wdyt?

Yes

Reply:
That's a great suggestion let's go with that.